libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -39,12 +39,50 @@ import {
   NumberString,
 } from './ui_strings';
 
-const Ballot = styled.div`
+export type BmdBallotPrintType = 'vxMark' | 'vsap';
+
+/**
+ * Max margin required to keep the ballot header visible when the page is
+ * partially covered by the VSAP infeed hood.
+ */
+export const MAX_VSAP_TOP_MARGIN = '1.75in';
```

Comment:
Similarly, maybe replace references to VSAP with VxMarkScan in constant names/comments

Reply:
+1